README.md

```diff
@@ -11,10 +11,10 @@ The focus of the currently released tool is on BPS and HVAC but we already provi
 ![Toolchain](docs/source/img/static/bim2sim_project_workflow_eng.png)
 
 ## Installation and Usage
-You can find detailed documentation and description how to install and to use in our [documentation](https://ebc.pages.rwth-aachen.de/EBC_all/github_ci/bim2sim/development/docs/overview.html). We recommend reading at least:
-* [Big Picture](https://ebc.pages.rwth-aachen.de/EBC_all/github_ci/bim2sim/development/docs/overview.html)
-* [Installation](https://ebc.pages.rwth-aachen.de/EBC_all/github_ci/bim2sim/development/docs/installation.html)
-* [First Steps](https://ebc.pages.rwth-aachen.de/EBC_all/github_ci/bim2sim/development/docs/first-steps.html)
+You can find detailed documentation and description how to install and to use in our [documentation](https://bim2sim.github.io/bim2sim//development/docs/overview.html). We recommend reading at least:
```

Comment:
this and the following urls dont work (also if  one of the twe "/" between bim2sim and development was deleted)

Reply:
thats because we need to merge this first, before the documentation of `development` branch becomes available. The CI builds the docs for the branch it's triggered on. So as long as this current state is not in `development` there will be no docs available, and the links won't work. Not really pretty, but should work.